katcp/kattypes.py

```diff
@@ -698,8 +698,7 @@ def raw_handler(self, msg):
                 new_args = unpack_types(types, msg.arguments, argnames, major)
                 return handler(self, *new_args)
 
-        raw_handler.__name__ = handler.__name__
```

Comment:
`wraps` is designed as a `@` decorator; this could (arguably) be more readably written as `update_wrapper(raw_handler, handler)`. Similarly for other cases where `wraps` is not used as a decorator.